index.bs

```diff
@@ -3726,6 +3788,38 @@ SHOULD be aborted.
     See [WHATWG HTML WG Issue #2711](https://github.com/whatwg/html/issues/2711) for more details.
 
 
+## WebAuthn Interfaces ## {#sctn-interfaces}
+
+The subection below defines custom interfaces used throughout WebAuthn.
+
+### Custom WebAuthn Exceptions ### {#iface-custom-webauthn-exceptions}
+
+For descriptions of these exceptions,
+please see [[#sctn-create-request-exceptions]] and [[#sctn-get-request-exceptions]].
+
+<xmp class="idl">
+[Exposed=Window, Serializable]
+interface HybridPrerequisitesError : DOMException {
+};
+
+[Exposed=Window, Serializable]
+interface TimeoutError : DOMException {
+};
```

Comment:
[Web IDL](https://webidl.spec.whatwg.org/#idl-DOMException-derived-interfaces) says:

>The [identifier](https://webidl.spec.whatwg.org/#dfn-identifier) of the [interface](https://webidl.spec.whatwg.org/#dfn-interface) must end with `Error`, and must not be any of the names in the [`DOMException` names table](https://webidl.spec.whatwg.org/#dfn-error-names-table).

So I don't think we need a separate `TimeoutError` since that's already covered by `DOMException`?

Reply:
Well that's convenient, I'll go ahead and take out the custom one I defined in this PR.